src/workingtitle-vcockpits-instruments/html_ui/Pages/VCockpit/Instruments/Shared/Map/WorkingTitle/MapInstrument.js

```diff
@@ -1215,44 +1217,18 @@ class MapInstrument extends ISvgMapRootElement {
     }
 	
     centerOnPlane() {
-        if (this.orientation == "north" || !this.offsetPlaneInHdgTrk) {
-			this.setNavMapCenter(this.navMap.planeCoordinates);
-			if (this.eBingMode == EBingMode.PLANE) {
-				this.airplaneIconElement.forceCoordinates(this.navMap.centerCoordinates.lat, this.navMap.centerCoordinates.long);
-			}
-		} else {
-			// MOD: if orientation is heading or track up, we want to place the plane 33% from the bottom of the map,
-			// but vector needs to be adjusted for overdraw factor of sqrt(2), which is where the magic number of 382 comes from (500-500/sqrt(2)+0.33*1000/sqrt(2))
-			let target = this.navMap.XYToCoordinatesFromPlaneWithRotation(new Vec2(500, 382));
-			this.setNavMapCenter(target);
-		}
+		// MOD: allow for arbitrary placement of plane
+		let r_x = this.navMap.aspectRatio < 1 ? this.navMap.aspectRatio : 1;
+		let r_y = this.navMap.aspectRatio > 1 ? 1 / this.navMap.aspectRatio : 1;
+		this.setNavMapCenter(this.navMap.XYToCoordinatesFromPlaneWithRotation(new Vec2(500 * (1 - r_x) + 1000 * (1 - this.planeTrackedPosX) * r_x, 500 * (1 - r_y) + 1000 * (1 - this.planeTrackedPosY) * r_y)));
```

Comment:
Could you do me a small favor and spread this across a couple lines to it's easier to read?   Sorry. :}

Reply:
Done. :)  e41ab0d8cb879829c1b3ee725b1e388c03090a44